2017/Team-Training/OMERO-web/index.html

```diff
@@ -0,0 +1,566 @@
+
+
+                <section>
+                    <h2>Relationship between apps</h2>
+
+                    <img style="height:500px" src="images/OmeroWeb-core-apps.png"/>
+
+                    <br>
+                    <a target="_blank" href="https://github.com/openmicroscopy/openmicroscopy/blob/develop/components/tools/OmeroWeb/omeroweb/">components/tools/OmeroWeb/omeroweb/</a>
+                    <!-- <ul>
+                        <li>webgateway, webclient, webadmin, feedback, api in
+                            <a target="_blank" href="https://github.com/openmicroscopy/openmicroscopy/tree/develop/components/tools/OmeroWeb/omeroweb">components/tools/OmeroWeb/omeroweb</a>.
+                        </li>
+                        <li>
+                            Configured in <a target="_blank" href="https://github.com/openmicroscopy/openmicroscopy/blob/develop/components/tools/OmeroWeb/omeroweb/settings.py">OmeroWeb/omeroweb/settings.py</a>
+                        </li>
+                        <li>OMERO.web core apps never installed independently.</li>
+                        <li>Django apps <i>CAN</i> link to or import from other installed apps</li>
+                        <li>
+                            webgateway provides <a target="_blank" href="http://www.openmicroscopy.org/site/support/omero5/developers/Web/WebGateway.html">core functionality</a>: 
+                            <ul>
+                                <li>Class LoginView extended by webclient & api</li>
+                                <li>html templates extended by webclient</li>
+                                <li>views.full_viewer() imported by webclient</li>
+                                <li>webgateway/imgData/id JSON used by webclient, figure, iviewer</li>
+                                <li>webgateway/render_image/id used by webclient, figure, iviewer</li>
+                            </ul>
+                        </li>
+                    </ul> -->
+                </section>
+
+
+                <section>
+                    <h2>OMERO.web for developers</h2>
+
+                    <ul>
+                        <li>clone openmicroscopy/openmicroscopy and build</li>
+                        <li>Run OMERO.web <a target="_blank" href="https://docs.openmicroscopy.org/latest/omero/developers/Web/Deployment.html">as described for developers</a>.
+                        </li>
+                        <li>Use nginx in development? Under discussion: <a target="_blank" href="https://trello.com/c/VdQ4HAl6/65-using-nginx-in-development">See trello card</a>.
+
+                        <li>Remove <code>dist/lib/python/omeroweb</code> and replace with sym-link to
+                            source at <code>components/tools/OmeroWeb/omeroweb</code>
+                        </li>
+
+                        <li>Here's what I do: 
+                            <a target="_blank" href="https://gist.github.com/will-moore/bbcb487b99502e9275c4fc6d80fb3b9c">
```

Comment:
It will preferable to adjust the script so a parameter can be specified so other can use it w/o any edit

Reply:
If this is considered useful to others and "the right way to do this" then it should be moved to the docs pages and fixed as you suggest. I've added a note to PR https://github.com/openmicroscopy/ome-documentation/pull/1799